pyBSDate/wrappers.py

```diff
@@ -42,10 +43,11 @@ def isoformat(self, lang='en'):
         return self.strftime(u"%Y-%m-%d", lang)
 
     def timetuple(self):
-        raise NotImplementedError
+        yday = self.toordinal() - datetime.date(self.year, 1, 1).toordinal() + 1
+        return time.struct_time((self.year, self.month, self.day, 0, 0, 0, self.weekday(), yday, -1))
 
     def toordinal(self):
-        raise NotImplementedError
+        return datetime.date(self.bs_year, self.bs_month, self.bs_day).toordinal()
```

Comment:
`datetime.date(2080, 02,30).toordinal()` will cause `ValueError: day is out of range for month`. since 02/30 makes sense in bs date but 02/30 in ad doesn't ?